### PR TITLE
Add "intervalSecs" to Generator db+api

### PR DESCRIPTION
**Pull request by iamigo**

### CODE REVIEW

File: tests/api/v1/generators/post.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ describe('tests/api/v1/generators/post.js >', () => {
 
       expect(res.body.apiLinks).to.be.an('Array');
       expect(res.body.name).to.include(generator.name);
+      expect(res.body).to.have.property('intervalSecs', 60);
```
Comment (jgraff2) [inline tests/api/v1/generators/post.js]: Should also have a test case where the value is actually set.

Comment (iamigo) [reply tests/api/v1/generators/post.js]: yes, done

File: api/v1/swagger.yaml [hunk 2]
```diff
@@ -11559,6 +11585,10 @@ definitions:
         readOnly: true
         description: >
           The collector id.
+      intervalSecs:
```
Comment (pallavi2209) [inline api/v1/swagger.yaml]: Do we need to add this field to GeneratorFieldsParam, GeneratorResponse and HeartbeatGeneratorResponse ? I added currentCollector to those.

File: tests/db/model/generator/create.js [hunk 3]
```diff
@@ -169,44 +170,6 @@ describe('tests/db/model/generator/create.js >', () => {
     });
   });
 
-  it('not ok, with both subjects and subjectQuery present in ' +
```
Comment (pallavi2209) [inline tests/db/model/generator/create.js]: Do we need to remove these 2 tests?

**Pull request merged**